packages/vue-pdf-viewer/src/components/PDFViewer.vue

```diff
@@ -3,6 +3,19 @@
         <div class="viewer-container" ref="viewerContainer">
             <div class="viewer"></div>
         </div>
+        <div v-if="newControlsEnabled" class="controls-new" ref="viewerControls">
```

Comment:
It think it would also make sense to do another round of package updates. There is a new version of pdfjs available for example.

Reply:
I did another round of updates and also added a new scale button for page-fit